packages/reactive-dict/reactive-dict.js

```diff
@@ -164,6 +164,23 @@ _.extend(ReactiveDict.prototype, {
     
   },
 
+  delete: function(key) {
+    var self      = this;
```

Comment:
Can we un-align this? We don't align assignments anywhere else in the code AFAIK